bot/cogs/help_channels.py

```diff
@@ -367,6 +370,18 @@ def get_used_names(self) -> t.Set[str]:
         log.info("Cog is ready!")
         self.ready.set()
 
+        self.report_stats()
+
+    def report_stats(self) -> None:
+        """Report the channel count stats."""
+        total_in_use = len(list(self.get_category_channels(self.in_use_category)))
+        total_available = len(list(self.get_category_channels(self.available_category)))
+        total_dormant = len(list(self.get_category_channels(self.dormant_category)))
```

Comment:
Since you just need the count, you could avoid creating a list each time the stats are reported with: `sum(1 for _ in self.get_category_channels(self.in_use_category))`. It wouldn't make an overly substantial difference though unless this was being called frequently.

Reply:
Addressed in 4adcb0b0ab2b8768e31043429bdfcbf4dab607e6